mappings/net/minecraft/world/chunk/Chunk.mapping

```diff
@@ -0,0 +1,129 @@
+CLASS net/minecraft/class_43 net/minecraft/world/chunk/Chunk
+	FIELD field_954 blocks [B
+	FIELD field_956 world Lnet/minecraft/class_18;
+	FIELD field_957 meta Lnet/minecraft/class_257;
+	FIELD field_958 skyLight Lnet/minecraft/class_257;
+	FIELD field_959 blockLight Lnet/minecraft/class_257;
+	FIELD field_960 heightmap [B
+	FIELD field_961 minHeightmapValue I
+	FIELD field_962 x I
+	FIELD field_963 z I
+	FIELD field_964 blockEntities Ljava/util/Map;
+	FIELD field_965 entities [Ljava/util/List;
+	METHOD <init> (Lnet/minecraft/class_18;II)V
+		ARG 1 world
+		ARG 2 x
+		ARG 3 z
+	METHOD <init> (Lnet/minecraft/class_18;[BII)V
+		ARG 1 world
+		ARG 2 blocks
+		ARG 3 x
+		ARG 4 z
+	METHOD method_858 (II)Z
+		ARG 1 x
+		ARG 2 z
+	METHOD method_859 getBlockId (III)I
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_860 setBlockId (IIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 rawId
+	METHOD method_861 setBlock (IIIII)Z
```

Comment:
what about setBlockIdAndMeta ? altho thats kinda long so i understand why you used setBlock even tho it implies it taking a Block as an argument

Reply:
I'd say `setBlockIdAndMeta` would be better as well.

Reply:
> what about setBlockIdAndMeta ? altho thats kinda long so i understand why you used setBlock even tho it implies it taking a Block as an argument

`setBlock` here is supposed to imply taking all data a block could possibly be identified with (id and meta) rather than a block instance.

And yeah, `setBlockIdAndMeta` is long. Add to that methods that cause block updates and you'll get `setBlockIdAndMetaWithUpdate` or something similar. I'd say we can sacrifice some descriptiveness in favour of nicer names here.

Reply:
ye i understand the use of a nicer name, i just wanted to raise a small discussion point